packages/auth/src/app.ts

```diff
@@ -231,6 +231,19 @@ export class App {
       grantRoutes.continue
     )
 
+    // Grant Cancel
+    this.publicRouter.delete(
+      '/continue/:id',
+      createValidatorMiddleware<ContinueContext>(openApi.authServerSpec, {
+        path: '/continue/{id}',
+        method: HttpMethod.POST
+      }),
+      this.config.bypassSignatureValidation
+        ? (ctx, next) => next()
+        : grantContinueHttpsigMiddleware,
```

Comment:
```suggestion
      grantContinueHttpsigMiddleware,
```
if :point_down: gets merged first
- https://github.com/interledger/rafiki/pull/879